post_office/management/commands/load_email_template.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management.base import BaseCommand
+from django.template import loader
+
+from ...models import EmailTemplate
+from ...logutils import setup_loghandlers
+
+logger = setup_loghandlers()
+
+
+def _get_template(template_name):
+    """Find the template and return the raw template file content"""
+
+    template = loader.get_template(template_name)
```

Comment:
Since we changed the template loading mechanism to a simple `open` instead of going through Django's loader, the exception no longer looks like this.

I think you can read the source by reading the file specified in `template_instance.origin.name` as suggested in this post: http://stackoverflow.com/questions/22256831/get-the-source-of-a-django-template

I don't mind either source reading mechanism. In your opinion, which kind of exception is better to show? Django's `TemplateDoesNotExist`, Python's `IOError` or should we make our own?